util/stopper.go

```diff
@@ -162,13 +162,20 @@ func (s *Stopper) SetStopped() {
 	}
 }
 
-// Quiesce moves the stopper to state draining, waits until all tasks
-// complete, then moves back to non-draining state. This is used from
+// Quiesce moves the stopper to state draining. Unquiesce() needs to
```

Comment:
You're right. This was unnecessary. I had somehow thought that it makes sense to move the state back to normal state before `Stop()`. Updated the comment.